CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.1.0/)
 ## [Unreleased 2.x](https://github.com/opensearch-project/flow-framework/compare/2.12...2.x)
 ### Features
 - Adding create ingest pipeline step ([#558](https://github.com/opensearch-project/flow-framework/pull/558))
+- adding create search pipeline step ([#569](https://github.com/opensearch-project/flow-framework/pull/569))
```

Comment:
Nit: we should probably be consistent with the verb tenses in these change logs.  We have "Add", "Added", "Adding", and now "adding" (lowercase).

Reply:
change the PR title, might be hard to enforce as from my experience these should match with PR titles

Reply:
Can you change it to `Adding` here as Dan mentioned?

Reply:
It should be `Added` though for CHANGELOG because we will use the same file for the release notes. So should be in the past tense here.